config/bootstrap.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+use Psr\Container\ContainerInterface;
+use Yiisoft\Translator\CategorySource;
+use Yiisoft\Translator\MessageFormatterInterface;
+use Yiisoft\Translator\MessageReaderInterface;
+use Yiisoft\Translator\TranslatorInterface;
+
+return [
+    'validator' => static function (ContainerInterface $container) {
```

Comment:
Actually it's not necessary. 
We need configure category and add it as reference in the [collection](https://github.com/yiisoft/demo/blob/master/config/params.php#L93).